packages/js/src/core/fetch-token.ts

```diff
@@ -10,7 +10,7 @@ export type FetchTokenByRefreshTokenParameters = {
   tokenEndPoint: string;
   refreshToken: string;
   resource?: string;
-  scope?: string[];
+  scopes?: string[];
 };
 
 type TokenSnakeCaseResponse = {
```

Comment:
TokenSnakeCaseResponse.id_token should be optional

References:

- [RefreshTokenTokenResponse](https://www.notion.so/silverhand/RefreshTokenTokenResponse-22b8a8cf606843df988141cec4e54cef)